tikv-client/src/main/java/com/pingcap/tikv/util/RangeSplitter.java

```diff
@@ -79,6 +83,25 @@ public String getHost() {
       return host;
     }
 
+    @Override
+    public boolean equals(Object another) {
+      if(!(another instanceof RegionTask)) return false;
+      RegionTask regionTask = (RegionTask) another;
+      return regionTask.host.equals(this.host)
+          && regionTask.ranges.equals(this.ranges)
+          && regionTask.region.equals(this.region)
+          && regionTask.store.equals(this.store);
+    }
+
+    @Override
+    public int hashCode() {
+      int hash = 7;
+      hash = 31 * hash * this.region.hashCode();
```

Comment:
I think you should use the following instead.
```suggestion
      hash = 31 * hash + this.region.hashCode();
```

Reply:
I already remove this. It seems we don't need this.